MAUI/MauiAppBasic/Platforms/Windows/App.xaml.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using MauiAppBasic.MSALClient;
+using Microsoft.UI.Xaml;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace MauiAppBasic.WinUI
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    public partial class App : MauiWinUIApplication
+    {
+        private const string RedirectURIWindows = "http://localhost";

```

Comment:
Preffer embedded webview on WIndows.

Reply:
This is cross platform. So, I want to keep one setting across all.